lib/vdsm/virt/vm.py

```diff
@@ -6169,7 +5939,7 @@ def merge(self, driveSpec, baseVolUUID, topVolUUID, bandwidth, jobUUID):
         return self._drive_merger.merge(
             driveSpec, baseVolUUID, topVolUUID, bandwidth, jobUUID)
 
-    def drive_get_actual_volume_chain(self, drive):
+    def query_drive_volume_chain(self, drive):
```

Comment:
Commit message: "it is needed for moving thing provisioning code out of VM class"

... thin provisioning ...